### ISSUE TITLE
Automatically deleting artifacts and models files from UI

### ISSUE THREAD

**Issue by montmejat** [allegroai/clearml-server#144]
Whenever I'm deleting a task using the web UI, it gives me the list of files that have not been deleted on my AWS storage:

![Screenshot from 2022-06-13 16-52-55](https://user-images.githubusercontent.com/29719028/173382819-e7a07709-ec7c-449d-80c9-a64ff200b3e7.png)

So each time I delete a task, I have to manually delete the files I have on AWS. Is there a way to delete tasks where it also deletes my files stored on my AWS bucket?

Thanks!

**Comment by aurelien-m** [comment]
By the way, it was a file added using:

```python
clearml_task.upload_artifact(...)
```

**Comment by jkhenning** [comment]
Hi @aurelien-m,

We're working on this right now - will update soon, I hope 🙂

**Comment by aurelien-m** [comment]
Thanks!

**Comment by Pierrelefort** [comment]
Any update on this ?

**Comment by ainoam** [comment]
The underlying mechanism for artifact deletion was reworked back in [ClearML Server v1.7](https://github.com/allegroai/clearml-server/releases/tag/v1.7.0), supporting deleting artifacts from the ClearML File server.
Support for external storage services (e.g. AWS S3) is still in the pipeline.

**Comment by TuMePJlaH** [comment]
Hello!
Do you have any update on this? I use ClearML Server 1.17 and still can't delete artifacts from AWS S3 when delete a task.

**Comment by ainoam** [comment]
@TuMePJlaH @Pierrelefort @montmejat Notice this was released as part of [v2.0](https://github.com/clearml/clearml-server/releases/tag/v2.0.0).
Let us know if there are any issues :)

**Comment by TuMePJlaH** [comment]
Yes, it works now! Thank you! I've been waiting for this feature for a long time!)
I think you have to add information about it into New Features on ClearML 2.0 release notes page)

**Comment by ainoam** [comment]
@TuMePJlaH It's there, just not identified with a link to this issue - we'll fix that 🙂
